nuxeo-easyshare-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-easyshare/nuxeo-easyshare-share-link.html

```diff
@@ -0,0 +1,100 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-easyshare-share-link`
+@group Nuxeo UI
+@element nuxeo-easyshare-share-link
+-->
+<dom-module id="nuxeo-easyshare-share-link">
+  <template>
+    <style>
+      :host {
+        display: inline-block;
+      }
+    </style>
+    <nuxeo-connection id="nxcon"></nuxeo-connection>
+
+    <template is="dom-if" if="[[_isAvailable(document)]]">
+      <paper-icon-button icon="[[icon]]" on-tap="_toggleDialog" noink></paper-icon-button>
+      <paper-tooltip>[[i18n('shareButton.tooltip')]]</paper-tooltip>
+    </template>
+
+    <nuxeo-dialog id="dialog" with-backdrop>
+      <h2>[[i18n('shareButton.dialog.heading')]]</h2>
+      <paper-input id="permalink" label="[[i18n('shareButton.copy.label', document.properties.dc:title)]]"
+                   value="[[_buildPermalink(document)]]"
+                   autofocus
+                   readonly>
+      </paper-input>
+      <template is="dom-if" if="[[_isEasyshare(document)]]">
+        <paper-input id="easyshareLink" label="[[i18n('easyshareFolder.share', document.properties.dc:title)]]"
```

Comment:
This label does not exist. It is [easysharefolder.share](https://github.com/nuxeo/nuxeo-web-ui/blob/master/i18n/messages.json#L382).